src/service_aggregator.py

```diff
@@ -170,7 +170,7 @@ async def entry(message, guid, coalesce_type, caller) -> (dict, int):
     except KeyError:
         return f"No query graph", 422
     results_cache = ResultsCache()
-    override_cache = (message.get("parameters") or {}).get("override_cache")
+    override_cache = message.get("bypass_cache", False)
```

Comment:
I think we need to keep the override_cache parameter in. I see these as two separate things. `bypass_cache` is for people who want fresh answers, but they aren't going to be giving a longer `timeout_seconds`, so we don't necessarily want this overriding the cache. I'd like to keep `override_cache` so we have some control over what's in there.

Reply:
I'm not sure I get it- they do the same thing, right?  You can use `bypass_cache` along with `timeout_seconds`.

Reply:
I'm saying I think they _should_ do different things. `bypass_cache` will be used by devs who want fresh answers and Aragorn should never use its cache. But users want good answers fast, so we have `override_cache` so that we can still fill the cache with good answers with the appropriate `timeout_seconds` for them.

Reply:
We also want to make sure that this flag is passed along to all underlying services, which I don't think it will do currently. There's another place further in the code that passes the `parameters` field along, so we should just be able to add this there.

Reply:
good point about passing the flag along

Reply:
But I still don't understand the difference you are proposing between `bypass` and `override`.   Is it that `bypass` doesn't write to the cache either?  Or that it ignores `timeout_seconds`?  Sorry for being obtuse.

Reply:
The flag should now make it to strider and multistrider

Reply:
Correct, `bypass_cache` shouldn't write to the cache either.

Reply:
I don't think that's the required behavior:
```
            Set to true in order to request that the agent obtain
            fresh information from its sources in all cases where
            it has a viable choice between requesting fresh information
            in real time and using cached information. The agent
            receiving this flag MUST also include it in TRAPI sent to
            downstream sources (e.g., ARS -> ARAs -> KPs).
```

Reply:
No, I think we would still adhere to the spec.
To give the full picture:
`bypass_cache`: We receive a query with this flag set to true. We don't use any caches in any of our tools. 
We run in real time and return fresh answers. Basically, we never touch the cache with this setting.
`override_cache`: We receive a query with this flag set to true AND `bypass_cache` is set to false. We don't grab from the cache and run in real time, but then we overwrite whatever was in the cache with this response. This will normally be run with an extended `timeout_seconds` to get the best results.

If we receive a query without either of these flags set, we use the cache whenever possible. This is the default behavior and will be used whenever someone sends a query through the UI.

Reply:
And why don't we want to write to the cache in the `bypass_cache` version?

Reply:
Because I don't envision anyone also setting `timeout_seconds` and getting the best results.